format.sh

```diff
@@ -0,0 +1,6 @@
+if [[ $(flutter format -n .) ]]; then
```

Comment:
we typically have these scripts in a 'scripts' or 'cmds' folder, what do you think?